website/.lycheeignore

```diff
@@ -0,0 +1,9 @@
+http://127.0.0.1:8000
+http://localhost:8000
+http://localhost:8089
+https://epubs.siam.org/doi/10.1137/0108018
+
```

Comment:
Suggest adding cache like https://github.com/paritytech/polkadot-sdk/blob/master/.config/lychee.toml#L4-L7 for example

Reply:
and CI https://github.com/paritytech/polkadot-sdk/blob/master/.github/workflows/check-links.yml